package.json

```
@@ -35,6 +35,7 @@
     "companion:build:android": "pnpm --filter @shelf/companion build:android",
     "companion:prebuild": "pnpm --filter @shelf/companion prebuild",
     "companion:prebuild:clean": "pnpm --filter @shelf/companion prebuild:clean",
+    "companion:test": "pnpm --filter @shelf/companion test",
```

Biome (2.5.7)

[error] The key companion:test was already declared.

(lint/suspicious/noDuplicateObjectKeys)